tests/RAGchain/preprocess/loader/test_hwp_table_loader.py

```diff
@@ -0,0 +1,23 @@
+import os
+import pytest
+
+from RAGchain.preprocess.loader.hwp_table_loader import HwpLoader
+
+file_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir, os.pardir, os.pardir, "resources",
```

Comment:
How about using pathlib.PurePath().parent instead of os.pardir. Like any other test cases?